docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -2952,11 +2954,11 @@ Create a volume
       "Name": "tardis",
       "Driver": "local",
       "Mountpoint": "/var/lib/docker/volumes/tardis",
-      "Status": null,
```

Comment:
Should this be removed? This was added in https://github.com/docker/docker/pull/21006 (https://github.com/docker/docker/commit/36a1c56cf555f8fe9ceabeebb8fc956e05863fc7); if the "local" driver doesn't return this, but other drivers do, we may want to update the example request/response to use a "custom" driver.

Perhaps it needs to be added to the "Inspect a volume" as well